src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -98,6 +98,7 @@ VALUES (1, 'Compliance'),
        (6, 'Proposed Restoration Activities'),
        (7, 'Verra Non-Permanence Risk Tool (NPRT)'),
        (8, 'Supplemental Files'),
+       (100, 'None'),
```

Comment:
IMO this seems very clumsy to me.

I do think the better approach, is to remove the "notify user for all category, if none is selected" behavior. Assuming that the number of global users that want to see all notifications is small, we can expect them to opt into each one instead. But we need product clarity for that.

Reply:
To guess at the original reasoning: if you forget to configure the categories for a user, getting all the notifications means it'll be obvious you forgot, whereas a lack of any notifications might take a lot longer to notice.